packages/inter-protocol/src/vpool-xyk-amm/multipoolMarketMaker.js

```diff
@@ -180,12 +180,16 @@ const start = async (zcf, privateArgs) => {
     privateArgs.storageNode,
     privateArgs.marshaller,
   );
-  const updateMetrics = () => {
+
+  const updateMetrics = () =>
     metricsPublication.updateState(
-      harden({ XYK: Array.from(secondaryBrandToPool.keys()) }),
+      harden({
+        XYK: Array.from(secondaryBrandToPool.keys()),
+        central: centralBrand,
+        poolFee: params.getPoolFee(),
+        protocolFee: params.getProtocolFee(),
```

Comment:
These are in `published.amm.governance` I'm pretty sure.

Reply:
Confirmed. I removed this file from the PR now anyway though.